stubs/tree-sitter/tree_sitter/__init__.pyi

```diff
@@ -13,5 +13,5 @@ class Language:
     language_id: int
     # library_path is passed into ctypes LoadLibrary
     def __init__(self, library_path: str, name: str) -> None: ...
-    def field_id_for_name(self, name): ...
-    def query(self, source): ...
+    def field_id_for_name(self, name: str) -> int | None: ...
+    def query(self, source: str) -> Incomplete: ...
```

Comment:
It returns a `tree_sitter.binding.Query` object from https://github.com/tree-sitter/py-tree-sitter/blob/master/tree_sitter/binding.c#L1389